src/parsers/xccdf.ts

```diff
@@ -131,15 +177,16 @@ export function processXCCDF(xml: string, removeNewlines: false, useRuleId: 'gro
       logger.warn(`Invalid value for extracted description: ${extractedDescription}`)
     }
 
-    control.impact = severityStringToImpact(rule['@_severity'] || 'medium', rule.group['@_id'])
-        
+    // control.impact = severityStringToImpact(rule['@_severity'] || 'medium', rule.group['@_id'])
```

Comment:
```suggestion
```

It looks like `rule.group['@_id']` was just used before to log a comment that if no severity was found that it would be set to a default of 0.5 for that id. Do we need this to be communicated to the user in another way? If so, maybe we can put it on the high level docs for this repo? Open to thoughts on if we need to at all.